internal/venice-common/src/main/java/com/linkedin/venice/pubsub/manager/TopicMetadataFetcher.java

```diff
@@ -373,6 +373,13 @@ long getLatestOffsetWithRetries(PubSubTopicPartition pubSubTopicPartition, int r
     }, retries, INITIAL_RETRY_DELAY, Duration.ofSeconds(5), Duration.ofMinutes(5), PUBSUB_RETRIABLE_FAILURES);
   }
 
+  CompletableFuture<Long> getLatestOffsetNoRetry(PubSubTopicPartition pubSubTopicPartition) {
+    return CompletableFuture.supplyAsync(() -> {
```

Comment:
Can we use `getLatestOffsetWithRetries` with 0 retries?

Reply:
you mean 1 attempt. I just wanted to remove the log algother.